### PR TITLE
Feature 174670147 dockerize and make deployable

### PR DESCRIPTION
**Pull request by vincecampanale**
@sandinmyjoints 
cc @mkalish 

**BEFORE MERGING**
- [x] Upload an initial image to ECR 
- [x] Upload staging secrets 
- [x] Set up staging fargate task definitions and services

### What/why

https://www.pivotaltracker.com/story/show/174670147

Added a Dockerfile, deploy scripts, and travis scripts for this repo so we can push an image to ECR and deploy to ECS. 

On merge, Travis will attempt to push a Docker image to ECR and update the staging sd-cors-anywhere service (running on Fargate) to run on a new revision of the existing staging-sd-cors-anywhere task definition which points to the new image tag.

New staging service here: https://console.aws.amazon.com/ecs/home?region=us-east-1#/clusters/sd-staging-fargate/services/staging-sd-cors-anywhere/tasks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,12 @@
 {
-  "name": "cors-anywhere",
-  "version": "0.4.3",
+  "name": "sd-cors-anywhere",
+  "version": "0.4.1",
```
Comment (vincecampanale) [inline package.json:3]: Didn't chance the `version` number intentionally. I didn't show up on my local diff.

**Comment by vincecampanale** [comment]
@sandinmyjoints From the logs, it looks like the service is able to start, but the task is cycling. I think it may have to do with the health check on the target group.

image

**Comment by vincecampanale** [comment]
There's also this error in the logs. Troubleshooting this too.
image

**Comment by vincecampanale** [comment]
Closing because the captcha script comes from Quizlet, not Heroku. 
https://www.pivotaltracker.com/story/show/174670147/comments/218783561

**Comment by sandinmyjoints** [comment]
Is this up and running on ECS, or not yet? @vincecampanale

**Comment by vincecampanale** [comment]
I had it up and running on ECS but took it down because it seemed like the plan was to not move forward with it. I can add it back.

**Comment by vincecampanale** [comment]
@sandinmyjoints There is now a `staging-sd-cors-anywhere` service in the `sd-staging-fargate` cluster available at sd-cors-anywhere.sdsandbox.com. GET requests through the OxyLabs proxy are currently being blocked, which I assume is due to the fact that the load balancer has not yet been whitelisted.

**Review by sandinmyjoints — APPROVED** [review]
:shipit:

**Pull request merged**